### PR TITLE
feat: add threeDSecure.cardinalSDKConfig to drop-in create

### PR DESCRIPTION
**Pull request by jplukarski**
### Summary

Add ability to pass in the `cardinalSDKConfig` object to the `dropin.create` call.

### Checklist

- [ x ] Added a changelog entry

- @crookedneighbor 
- @jplukarski

### CODE REVIEW

**Review by sshropshire — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -134,11 +134,18 @@ var VERSION = '__VERSION__';
  */
 
 /**
- * @typedef {object} threeDSecureOptions _Deprecated_ If the `threeDSecureOptions` passed into the create call is an object, you may set the `amount` to verify with 3D Secure. However, it's recomended that you pass `true` instead of a configuration object and do all 3D Secure configuration in the {@link Dropin#requestPaymentMethod|requestPaymentMethod options}.
```
Comment (sshropshire) [inline src/index.js:137]: random q: does this essentially mean the `threeDSecureOptions` parameter has been "un-deprecated" here?

Comment (jplukarski) [reply src/index.js:137]: Yep! We are "un-deprecating" it since now merchants can pass in a `cardinalSDKConfig` object here.

Comment (crookedneighbor) [reply src/index.js:137]: yeah, originally, all the 3ds config stuff was put in an object at the time of creation (at the time, just `amount`). With 3ds 2, most of the parameters are things you collect _as_ you check out, so you don't have them at the point of creating Drop-in, so we elected to pass those values at the time you call `requestPaymentMethod`.

With this addition, we need it at the time of creation, so we're un-deprecating the object as a whole, and only deprecating the `amount` field.

File: src/index.js [hunk 2]
```diff
@@ -134,11 +134,18 @@ var VERSION = '__VERSION__';
...
+ * @typedef {object} threeDSecureOptions If the `threeDSecureOptions` passed into the create call is an object, you may set the `amount` to verify with 3D Secure. However, it's recomended that you pass `true` instead of a configuration object and do all 3D Secure configuration in the {@link Dropin#requestPaymentMethod|requestPaymentMethod options}.
  *
  * @param {string} amount The amount to verify with 3D Secure.
  */
 
+/**
+ * @typedef {object} threeDSecureOptions Configuration options to pass when creating the 3D Secure module used in Drop-in.
```
Comment (sestevens) [inline src/index.js:143]: Why do we have `threeDSecureOptions` twice? Should the first one that only lists `amount` as a param be removed in favor of this one?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ CHANGELOG
 unreleased
 ----------
 - Update braintree-web to v3.76.4
+- Add `threeDSecure.cardinalSDKConfig` option to `dropin.create`
```
Comment (sestevens) [inline CHANGELOG.md:7]:
```suggestion
- Add `threeDSecureOptions.cardinalSDKConfig` option to `dropin.create`
```

Comment (crookedneighbor) [reply CHANGELOG.md:7]: It's:

```
braintree.dropin.create({
  threeDSecure: {
    cardinalSDKConfig: { /* options */ }
  }
```

`threeDSecureOptions` is a type definition in the JSDOC, so we can link to it in the options from `create`. See the `threeDSecure` option here: https://braintree.github.io/braintree-web-drop-in/docs/current/module-braintree-web-drop-in.html#.create

File: src/index.js (hunk 2 above)
Comment (crookedneighbor) [reply src/index.js:143]: woah, yeah, that is a mistake. We intended to add an additional param here. Not sure how we ended up copying the whole thing. Thanks for the heads up.

File: CHANGELOG.md (hunk 3 above)
Comment (sestevens) [reply CHANGELOG.md:7]: Gotcha, that makes sense 👍

**Review by sestevens — APPROVED** [review]

**Pull request merged**